### PR TITLE
add rspec

### PR DESCRIPTION
**Pull request by demands14**
## Related Tickets
- [#53737](https://edu-redmine.sun-asterisk.vn/issues/53737)

## WHAT (optional)
- write rspec for: 
TicketsController
StaticPagesController
ShowsController
RatingsController
PaymentsController
PaymentActivationsController
OrderHistorysController
MoviesController
FavoritesController

## HOW
- Write rspec test for controller

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by demands14** [comment]
ready

File: spec/controllers/ratings_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    #   before{post :create, format: :js, params: {id: movie.id, rating: {comment: "this is comment"}}}
+
+    #   it "assigns the request create comment" do
+    #     expect(assigns(:rating)).to eq([rating])
+    #   end
+
+    #   it "respond to js" do
+    #     expect(response).to be_successful
+    #   end
+    # end
```
Comment (ghost) [inline spec/controllers/ratings_controller_spec.rb]: sao laij cmt thees nayf em

**Review by ghost — COMMENTED** [review]
cmt

File: spec/controllers/ratings_controller_spec.rb (hunk 1 above)
Comment (demands14) [reply spec/controllers/ratings_controller_spec.rb]: em sửa rồi anh ạ

**Comment by demands14** [comment]
Ready

File: spec/controllers/movies_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    context "when movie is not present" do
+      before{get :show, params: {id: 0}}
+
+      it "show flash message" do
+        expect(flash[:warning]).to eq I18n.t("not_found")
+      end
+    end
+  end
+
+  describe "GET #sort" do
```
Comment (huongnt-2545) [inline spec/controllers/movies_controller_spec.rb:30]: bổ sung thêm test case cho trường hợp sắp xếp

**Comment by demands14** [comment]
Ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**